pages/journal.js

```diff
@@ -3,16 +3,320 @@ import Helmet from 'react-helmet';
+        year: 'Spring 2012',
+		cover: '/assets/2012cover.png',
+        content: [
+			  {
+				  author: "Dawei Ding",
+				  title: "A Sociopolitical and Economic Analysis of Recent Shanghai Policies on Hukou and Migrant Worker Insurance"
+			  }, {
+				  author: "Tara Yarlagadda",
+				  title: "An Analysis of State-based Narratives in Pakistani and Indian Textbooks"
+			  }, {
+				  author: "Muyang Chen",
+				  title: "The Current Buraku Issue in Tokyo and Its Solutions"
+			  }, {
+				  author: "Benjamin Knight",
+				  title: "Deconstructing Japan's Military Globalism: The March to \"Normal\" Nationhood"
+			  }
+		  ]
+      }, {
+		title: 'BSJAS Inaugural Edition',
+        year: 'Fall 2010',
+		cover: '/assets/2010cover.png',
+        content: [
+			  {
+				  author: "James Lin",
+				  title: "Soviet Internationalism, Chinese Nationalism, and Early Kuomintang Struggles"
+			  }, {
+				  author: "Alessandro Tiberio",
+				  title: "Becoming Taiwanese"
+			  }, {
+				  author: "Andrea Horbinski",
+				  title: "The Tokyo Tribunal"
+			  }, {
+				  author: "Nina Tompkin",
+				  title: "Professor Interview featuring Lanchih Po"
+			  }
+		  ]
+      },
+	  
+    ];
+
+    const handleClick = (index) => {
+      this.setState({
+        activeTab: index,
+      });
+    }
+
+    const tabs = () => {
+      let tabs = [{title: 'Spring 2015'}, {title: 'Spring 2014'}, {title: 'Spring 2013'}, {title: 'Spring 2012'}, {title: 'Fall 2010'}];
+      const HTMLify = function (item, i) {
+        return (
+			<div
+			  key={i}
+			  onClick={() => handleClick(i)}
+			  className = "tab"
+			>
+			  {item.title}
+			</div>
+        );
+      }
+      return (
+		<div className="tab-nav">
+		  {tabs.map(HTMLify)}
+		</div>
+	  );
+    }
+
+    const tabContent = () => {
+      const activeTab = this.state.activeTab;
+      const data = tabData[activeTab];
+	  const content = function (item) {
+		return (
+		  <div>
+			<div className="author">
+			  {item.author}
+			</div>
+			
+			<div className="title">
+			  {item.title}
+			</div>
+		  </div>
+		);
+	  }
+      return (
+        <div className="tab-content">
+		  <div className="tab-title">
+		    {data.title}
+		  </div>
+		  
+		  	<img className="cover-img" src={data.cover} />
+
+		  
+		  {data.content.map(content)}
+		</div>
+      );
+    }
+	
     return (
       <div className="">
         <Helmet
           title={config.siteTitle}
         />
-        <h1>
-          Journal
-        </h1>
-      </div>
+		
+		<div className="content-header">
+          <span>Journal / </span>
+          <span className="subheader">Recent Publications</span>
+        </div>
+		
+		<div className="journals">
+		  {tabs()}
+		  {tabContent()}
+		</div>
+		
+        <div className="submit">
+		  <div className="content-header">
+            <span>Journal / </span>
+            <span className="subheader">Submit</span>
+          </div>
+		
+		  <div className="question">
```

Comment:
Take the QA data and put it in its own static file config file like above, then you can import it and map over the data to generate this